projects/extension/src/background/ConnectionManager.ts

```diff
@@ -156,10 +156,17 @@ export class ConnectionManager<SandboxId> {
     chainName: string,
     maxUtf8BytesSize?: number,
   ): Promise<string | undefined> {
+    let chain: SmoldotChain | undefined = undefined
     try {
-      return await this.#wellKnownChains
-        .get(chainName)!
-        .chain.databaseContent(maxUtf8BytesSize)
+      chain = this.#wellKnownChains.get(chainName)!.chain
+    } catch (error) {
+      // If an exception is thrown it means that the specific chainName is not among the
+      // #wellknownchains (either did not start or was disconnected). This is separated from
+      // the error below as it does not show a crash in smoldot but that a chain does not exist
```

Comment:
```suggestion
      // If an exception is thrown it means that the specific chainName is not among the
      // #wellknownchains. This is separate from the error below as it does not show a
      // crash in smoldot but that a chain does not exist
```

Well-known chains can't be "disconnected", there's no way to do that